meshtastic/supported_device.py

```diff
@@ -205,24 +250,39 @@ def __init__(
     usb_vendor_id_in_hex="1a86",
     usb_product_id_in_hex="55d4",
 )
+nano_g1_explorer = SupportedDevice(
+    name="Nano G1 Explorer",
+    version="",
+    for_firmware="nano-g1-explorer",
+    baseport_on_linux="ttyACM",
+    baseport_on_mac="cu.usbmodem",
+    usb_vendor_id_in_hex="1a86",
+    usb_product_id_in_hex="55d4",
+)
 
 supported_devices = [
     tbeam_v0_7,
     tbeam_v1_1,
     tbeam_M8N,
     tbeam_M8N_SX1262,
+    tbeam_S3_core,
     tlora_v1,
     tlora_v1_3,
     tlora_v2,
     tlora_v2_1_1_6,
+    tlora_t3s3,
     heltec_v1,
     heltec_v2_0,
     heltec_v2_1,
+    heltec_v3,
+    heltec_wsl_v3,
     meshtastic_diy_v1,
     techo_1,
     rak4631_5005,
     rak4631_5005_epaper,
     rak4631_19003,
     rak11200,
```

Comment:
The RAK targets seem to be out of sync too. We dropped the distinction 5005/19003 a while ago.

Reply:
Do we need to update this list?